doc/faq.md

```diff
@@ -56,7 +56,12 @@ USER 1001
 If you aren't sure what dependencies are required, start up a container using the image in the `FROM` line as root. That will look something like this.
 `docker run -u 0 -it --rm --entrypoint /bin/bash quay.io/operator-framework/ansible-operator:<sdk-tag-version>`
 
+## My cache might be stale if I read from a cache! How should I deal with that?
```

Comment:
> We should not put every upstream FAQ in this doc, only certain ones that users are likely to encounter in common constructs ex. the reconcile loop, like this one.

@estroz @theishshah Or better yet. Put a section at the very top of this FAQ doc that says something like:
```
## Controller Runtime FAQ

Please see the upstream Controller Runtime FAQ first for any questions related to runtime mechanics or controller-runtime APIs. 
```


Otherwise with this we'd have to keep adding more questions verbatim just to say `Please see the Controller Runtime FAQ`.